src/core/Akka/Actor/ActorBase.cs

```diff
@@ -147,6 +147,17 @@ protected static IActorContext Context
             }
         }
 
+        private Scheduler.TimerScheduler timers;
+        public Scheduler.ITimerScheduler Timers
+        {
+            get
+            {
+                if (timers == null)
+                    timers = new Scheduler.TimerScheduler(Context);
```

Comment:
This can be also written as `get => timers ?? (timers = new Scheduler.TimerScheduler(context))`

Reply:
changed